src/e2e-test/java/io/cdap/plugin/common/stepsdesign/TestSetupHooks.java

```diff
@@ -354,7 +360,8 @@ private static void createSourceBQTableWithQueries(String bqCreateTableQueryFile
       // Iterator on TableResult values in getSoleQueryResult method throws NoSuchElementException
```

Comment:
Remove the comment from here.

Reply:
done
commit id- https://github.com/cloudsufi/google-cloud/commit/defc54df110255ef539dea8d206481cc145b2fec